Shared/Samples/Show service area/ShowServiceAreaView.swift

```diff
@@ -0,0 +1,215 @@
+    
+    var body: some View {
+        MapView(map: model.map, graphicsOverlays: [
+            model.facilitiesGraphicsOverlay,
+            model.barriersGraphicsOverlay,
+            model.serviceAreaGraphicsOverlay
+        ])
+        .onSingleTapGesture { _, point in
+            model.placeGraphicOnTapLocation(point: point, selection: selectedGraphicType)
+        }
+        .toolbar {
+            ToolbarItemGroup(placement: .bottomBar) {
+                Picker("Mode", selection: $selectedGraphicType) {
+                    ForEach(SelectedGraphicType.allCases, id: \.self) {
+                        Text($0.label)
+                    }
+                }
+                .pickerStyle(.segmented)
+                Spacer()
+                Menu {
+                    Slider(value: $model.secondTimeBreak, in: 1...10, step: 1) {
+                        Text("Finished: \(Int(model.secondTimeBreak))")
+                    }
+                    Slider(value: $model.firstTimeBreak, in: 1...10, step: 1) {
+                        Text("Start: \(Int(model.firstTimeBreak))")
+                    }
+                } label: {
+                    Label("Time", systemImage: "gear")
+                }
+                Spacer()
+                Button("Service Area") {
+                    Task {
+                        do {
+                            try await model.showServiceArea()
+                        } catch {
+                            self.error = error
+                        }
+                    }
+                }
+                Spacer()
+                Button("Clear", systemImage: "trash.fill") {
+                    model.removeAllGraphics()
+                }
+            }
+        }
+        .errorAlert(presentingError: $error)
+    }
+}
+
+private enum SelectedGraphicType: Equatable, CaseIterable {
+    case facilities, barriers
+    
+    /// The string representation of the SelectedGraphic type.
+    var label: String {
+        switch self {
+        case .barriers: "Barriers"
+        case .facilities: "Facilities"
+        }
+    }
+}
+
+private extension ShowServiceAreaView {
+    @MainActor
+    class Model: ObservableObject {
+        /// Map with terrain style centered over San Diego.
+        let map: Map = {
+            let map = Map(basemapStyle: .arcGISTerrain)
+            map.initialViewpoint = Viewpoint(
+                center: Point(
+                    x: -13041154,
+                    y: 3858170,
+                    spatialReference: .webMercator
+                ),
+                scale: 60000
+            )
+            return map
+        }()
+        
+        private(set) var facilitiesGraphicsOverlay: GraphicsOverlay = {
+            var facilitiesGraphicsOverlay = GraphicsOverlay()
+            // Previously using PictureMarkerSymbol(image: UIImage(named: "Facility")!)
+            let facilitySymbol = PictureMarkerSymbol(image: UIImage(named: "PinBlueStar")!)
```

Comment:
@yo1995 the PinBlueStar looks almost exactly like the Facility icon, should I keep it or use the one from the previous sample?

Reply:
Blue pin is fine